docs/manual/related-projects.md

```diff
@@ -63,7 +63,7 @@ awareness of BTRFS in dpkg/rpm itself) will be required.
 The OSTree author believes that having total freedom at the block
 storage layer is better for general purpose operating systems. For
 example, with OSTree, one is free to use BTRFS in any way you like -
-you can use a subvolume for `/home`, or you cannot.
+you can use a subvolume for `/home`, or you can not.
```

Comment:
this will break make syntax-check.  Fix for me if you add this part:

diff --git a/cfg.mk b/cfg.mk
index 3806209..d6a5039 100644
--- a/cfg.mk
+++ b/cfg.mk
@@ -1,4 +1,4 @@
-export VC_LIST_EXCEPT_DEFAULT=^(git.mk|lib/.*|m4/.*|md5/.*|build-aux/.*|src/gettext\.h|.*ChangeLog|buildutil/.*)$$
+export VC_LIST_EXCEPT_DEFAULT=^(docs/.*|git.mk|lib/.*|m4/.*|md5/.*|build-aux/.*|src/gettext\.h|.*ChangeLog|buildutil/.*)$$
 

and we skip the syntax checks on the documentation.